<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>614</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>588</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>614</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>猫猫翻金币</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/res/Coin0001.png</normaloff>:/res/Coin0001.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>588</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>588</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainscene">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#mainscene
{
	border-image: url(:/res/OtherSceneBg.png);
}</string>
       </property>
       <widget class="QPushButton" name="start_button">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>400</y>
          <width>131</width>
          <height>121</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>180</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border:0px;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/res/MenuSceneStartButton.png</normaloff>:/res/MenuSceneStartButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="title">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>201</width>
          <height>71</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/res/Title.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="LevelScene" name="levelscene">
       <property name="styleSheet">
        <string notr="true">#levelscene{
border-image: url(:/res/OtherSceneBg.png);}</string>
       </property>
       <widget class="MyButton" name="back_button">
        <property name="geometry">
         <rect>
          <x>243</x>
          <y>556</y>
          <width>77</width>
          <height>32</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border:0px;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/res/BackButton.png</normaloff>:/res/BackButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>77</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="PlayScene" name="playscene">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">#playscene{
	border-image: url(:/res/PlayLevelSceneBg.png);
}</string>
       </property>
       <widget class="MyButton" name="back_button_2">
        <property name="geometry">
         <rect>
          <x>246</x>
          <y>558</y>
          <width>77</width>
          <height>32</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{border:0px;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/res/BackButton.png</normaloff>:/res/BackButton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>77</width>
          <height>32</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="save_button">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>560</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(168, 0, 0);</string>
        </property>
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>新建</string>
    </property>
    <addaction name="actionquit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionquit">
   <property name="text">
    <string>quit</string>
   </property>
   <property name="iconText">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header location="global">mybutton.h</header>
  </customwidget>
  <customwidget>
   <class>LevelScene</class>
   <extends>QWidget</extends>
   <header location="global">levelscene.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayScene</class>
   <extends>QWidget</extends>
   <header location="global">playscene.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
